packages/wpcom-template-parts/src/universal-footer-navigation/index.tsx

```diff
@@ -472,7 +473,14 @@ const UniversalNavbarFooter = ( {
 	const isEnglishLocale = useIsEnglishLocale();
 	const pathNameWithoutLocale =
 		currentRoute && removeLocaleFromPathLocaleInFront( currentRoute ).slice( 1 );
-	const automatticBranding = useAutomatticBrandingNoun();
+	const [ automatticBranding, setAutomatticBranding ] = useState( { article: '', noun: '' } );
+	const currentAutomatticBranding = useAutomatticBrandingNoun();
+
+	useLayoutEffect( () => {
+		setAutomatticBranding( currentAutomatticBranding );
+		// Automattic Branding should be set only once after the page load
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [] );
```

Comment:
Here a new random slogan is generated on every render, but used only once in a mount effect. That's weird and the ESLint warning warns about that weirdness.

It would be better to extract the `useTranslate` function out of `useAutomatticBrandingNoun`, and `useTranslate` was also the only reason why it was a hook:
```js
const translate = useTranslate();
useEffect( () => {
  setAutomatticBranding( getAutomatticBrandingNoun( translate ) );
}, [ translate ] );
```

Reply:
Updated at https://github.com/Automattic/wp-calypso/pull/77916/commits/1425f4aefe75704dba82e9839706210f7c355ff2